src/api/application-api.md

```diff
@@ -96,7 +96,7 @@ app.directive('my-directive', {
   mounted() {},
   // called before the containing component's VNode is updated
   beforeUpdate() {},
-  // called after the containing component's VNode and the VNodes of its children // have updated
+  // called after the containing component's VNode and the VNodes of its children have updated
```

Comment:
```suggestion
  // called after the containing component's VNode and the VNodes of its
  // children have updated
```

I think this may have been mucked up by an auto-formatter. I believe that comment is supposed to be split over two lines so that it doesn't trigger a horizontal scrollbar.